Fits/Electronic Attack Frigates/[ADV-30] Hyena.md

```diff
@@ -0,0 +1,34 @@
+# Hyena
+
+Advanced EWAR Frigate
+
+## Description
+
+Utilizes Target Painters to enhance the friendly fleet's DPS against a hostile target. This ship is entirely reliant on range to avoid incoming DPS so do not burn close unless absolutely necessary. Use 3x TP if purely fleet support, mix in webs if it's going to be possible to get close enough to apply them.
+
+## Fit
+```
+[Hyena, [ADV-30] Hyena]
```

Comment:
Hyena should definitely be shield and speed, I think?

Reply:
Goal was really just to maximize the EWAR effectiveness of these and maximize range, this thing operates comfortably at like 130km so I think tank is kind of secondary. If it gets caught 1k EHP isn't going to save it realistically and you lose a lot of what it is actually on grid to do.

Reply:
Fixed, kept the 130km range but got some more speed/agility.